### PR TITLE
Standardize Quit on Linux/Windows

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

@MarcoPolo, this PR will affect Windows too, do you think it does the right thing there?

### CODE REVIEW

File: desktop/app/menu-helper.js [hunk 1]
```diff
@@ -58,7 +58,10 @@ export default function makeMenu (window) {
   } else {
     const template = [{
       label: '&File',
-      submenu: [{label: '&Close', accelerator: 'CmdOrCtrl+W', role: 'close'}],
+      submenu: [
+       {label: '&Close', accelerator: 'CmdOrCtrl+W', role: 'close'},
+       {label: '&Quit ', accelerator: 'CmdOrCtrl+Q', role: 'close'},
```
Comment (MarcoPolo) [inline desktop/app/menu-helper.js]: does this just close a window?

Comment (chromakode) [inline desktop/app/menu-helper.js]: Is there intended to be a space after "&Quit"?

Comment (cjb) [inline desktop/app/menu-helper.js]: @MarcoPolo Yep!

@chromakode Nope!

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 3c54cc36fe0a...9c878b1dc01c introduced some visual changes on win32:
 * Changed: **user_search-Normal** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1468356020752-3c54cc36fe0a-9c878b1dc01c-win32/user_search-Normal-3c54cc36fe0a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1468356020752-3c54cc36fe0a-9c878b1dc01c-win32/user_search-Normal-9c878b1dc01c.png)  
   user_search-Normal comparison
 * Changed: **Search_User_Pane-Broken_tracker_not_following** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1468356020752-3c54cc36fe0a-9c878b1dc01c-win32/Search_User_Pane-Broken_tracker_not_following-3c54cc36fe0a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1468356020752-3c54cc36fe0a-9c878b1dc01c-win32/Search_User_Pane-Broken_tracker_not_following-9c878b1dc01c.png)  
   Search_User_Pane-Broken_tracker_not_following comparison

**Comment by cjb** [comment]
@MarcoPolo changed to close all windows, thanks!

**Comment by MarcoPolo** [comment]
👍

**Comment by chromakode** [comment]
:+1:

**Pull request merged**